src/components/ConnectWallet/ConnectWalletModal.tsx

```diff
@@ -80,6 +87,21 @@ export const DisconnectWalletModal: FC<DisconnectWalletModalProps> = ({
   const { disconnect } = useDisconnect()
   const [isCopied, setCopied] = useState(false)
   const { chain } = useNetwork()
+  const resetAfterMs = 500
```

Comment:
I pulled this down for a test and think this would feel better with 1000ms. What do you think?